client/src/modules/tags/tags.html

```diff
@@ -0,0 +1,40 @@
+<div class="flex-header">
+    <div class="bhima-title">
+      
+      <ol class="headercrumb">
+        <li class="static" translate>TREE.ADMIN</li>
+        <li class="title" translate>TREE.TAGS</li>
+      </ol>
+  
+      <div class="toolbar">
+        <div class="toolbar-item">
+          <button class="btn btn-default" ng-click="TagsCtrl.createUpdateTagsModal()" data-method="create">
+            <span class="fa fa-plus"></span>
+            <span translate class="text-capitalize">FORM.LABELS.ADD</span>
+          </button>
+        </div>
+        <bh-filter-toggle on-toggle="TagsCtrl.toggleInlineFilter()">
+        </bh-filter-toggle>
+      </div>
+    </div>
+  </div>
+  
+  <!-- main content -->
+  <div class="flex-content">
+    <div class="container-fluid">
+      <div
+        id="tags-grid"
+        class="grid-full-height"
+        ui-grid="TagsCtrl.gridOptions"
+        ui-grid-auto-resize
+        ui-grid-resize-columns
+        ui-grid-move-columns>
+  
+        <bh-grid-loading-indicator
+          loading-state="TagsCtrl.loading"
+          empty-state="TagsCtrl.gridOptions.data.length === 0">
```

Comment:
Can you include an `error` state here in case something goes wrong?  Often, things like HTTP errors will happen and the user won't know what went wrong.